CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [5.3.1](https://github.com/ubuntu/yaru.dart/compare/v5.3.0...v5.3.1) (2024-10-27)
```

Comment:
This has the same version number as the previous entry

Reply:
Okay I think I can fix this